### ISSUE TITLE
Error on uploading base64 image

### ISSUE THREAD

**Issue by sumit10** [balderdashy/skipper#112]
Hi I am getting error on uploading base64 image 
```
Unable to parse HTTP body- error occurred :: 'Error: EUNFNTEX: Timed out waiting for known text parameters to finish streaming their bytes into the server.\n at setTimer (/app/node_modules/sails/node_modules/skipper/lib/Parser/prototype.parseReq.js:179:23)\n at null. (/app/node_modules/sails/node_modules/skipper/node_modules/async/lib/async.js:758:21)\n at null._onTimeout (/app/node_modules/sails/node_modules/skipper/node_modules/async/lib/async.js:173:37)\n at Timer.listOnTimeout (timers.js:119:15)'
```

**Comment by Ryanc1256** [comment]
I got this too, but it wasn't a  base64 image... it was a image...

**Comment by sailsbot** [comment]
Thanks for posting, @sumit10.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by magicdawn** [comment]
so what happens ? here is also a image

**Comment by Eklado** [comment]
Hello,
I get the same error on uploading base64 image.
I found only one solution for this issue:

http://www.howtobuildsoftware.com/index.php/how-do/bkPE/nodejs-base64-sailsjs-data-uri-data-url-sailsjs-upload-huge-base64-image

It's working on normal servers, BUT this solution not possible on Heroku, since I can't access the server code.
So did anyone found a better solution for this issue?
it is preferable to set the request timeout value in the config files

Thank you in advance
